app/src/main/java/no/usn/mob3000/ui/screens/auth/LoginScreen.kt

```diff
@@ -92,6 +93,7 @@ fun LoginScreen(
                     onValueChange = { inputEmail = it },
                     label = { Text(stringResource(R.string.auth_login_email)) },
                     placeholder = { Text(stringResource(R.string.auth_login_email_placeholder)) },
+                    textStyle = TextStyle(color = MaterialTheme.colorScheme.onBackground),
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).